invenio_rdm_records/records/dumpers/pids.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""DataCite-based data model for Invenio."""
```

Comment:
```suggestion
"""ElasticSearch dumpers for pids information."""
```
I think the module description is wrong also in the other dumper files.

Reply:
Yes, fixing them. Nice catch!